src/providers/incoming-data/incoming-data.ts

```diff
@@ -1561,20 +1574,13 @@ export class IncomingDataProvider {
         // call IAB and attempt pairing
         case 'pairingRequired':
           const authRequiredInfoSheet = this.actionSheetProvider.createInfoSheet(
-            'auth-required'
+            'pairing-required'
           );
           await authRequiredInfoSheet.present();
-          authRequiredInfoSheet.onDidDismiss(() => {
-            this.iabCardProvider.show();
-            setTimeout(() => {
-              this.iabCardProvider.sendMessage(
-                {
-                  message: 'pairingOnly',
-                  payload: { paymentUrl: data }
-                },
-                () => {}
-              );
-            }, 100);
+          authRequiredInfoSheet.onDidDismiss(async () => {
+            await this.externalLinkProvider.open(
+              `https://${host}/id/verify?context=unlockv&id=${invoiceId}`
```

Comment:
need to fix host here - host does not have link. subdomain removed so it does not redirect correctly